examples/morpheusvm/controller/controller.go

```diff
@@ -92,6 +90,8 @@ func (*factory) New(
 			err,
 		)
 	}
+	c.genesis.ChainID = chainID
+	c.genesis.NetworkID = networkID
```

Comment:
I think the `chainID` and `networkID` should also be parameters to constructing the genesis instead of setting them here